packages/ag-charts-website/update-algolia.js

```diff
@@ -151,8 +150,8 @@ const createRecords = async (browser, url, framework, breadcrumb, rank, loadFrom
             objectID: hashPath,
             breadcrumb,
             title,
-            heading: heading ? heading.replaceAll('\n    ', '').replaceAll('\n', '') : undefined,
-            subHeading: subHeading ? subHeading.replaceAll('\n    ', '').replaceAll('\n', '') : undefined,
+            heading: heading ? heading.replaceAll('\n    ', '').replaceAll('\n', '').trim() : undefined,
```

Comment:
Do you need the `replaceAll` stuff if you're using `trim()`? Looks like both get rid of whitespace?